test/cpp/aot_inductor/test.cpp

```diff
@@ -26,18 +26,18 @@ void test_aoti(const std::string& device) {
   std::string inputs_attr = "inputs_" + device;
   std::string outputs_attr = "outputs_" + device;
   const auto& model_so_path = data_loader.attr(path_attr.c_str()).toStringRef();
-  const auto& input_tensors =
+  auto input_tensors =
       data_loader.attr(inputs_attr.c_str()).toTensorList().vec();
   const auto& ref_output_tensors =
       data_loader.attr(outputs_attr.c_str()).toTensorList().vec();
 
   std::unique_ptr<torch::inductor::AOTIModelContainerRunner> runner;
   if (device == "cuda") {
     runner = std::make_unique<torch::inductor::AOTIModelContainerRunnerCuda>(
-        model_so_path.c_str());
+        model_so_path);
   } else if (device == "cpu") {
     runner = std::make_unique<torch::inductor::AOTIModelContainerRunnerCpu>(
-        model_so_path.c_str());
+        model_so_path);
   } else {
     testing::AssertionFailure() << "unsupported device: " << device;
```

Comment:
Feels like we should just port this test into `test_aot_inductor.py` eventually.